### DISCUSSION TITLE
Navbar still showing in mobile

### DISCUSSION THREAD

**Discussion by meetmarvin** [kdevo/osprey-delight#27] [Q&A] [answered]
Hi,

I noticed that the navbar is still showing at the bottom when using the mobile version. I checked the showcase sites and they are both working fine. Is there a config that I need in order to hide the navbar when mobile version is loaded and at the top most of the page (I'm expecting to only see logo and background image)? 

Not sure if I explained it well. I can send a screen capture/recording if you want.

Thank you!

**Comment by kdevo** [comment, marked as answer]
Hi, 

both sites use custom SCSS (see [CUSTOMIZING docs](https://github.com/kdevo/osprey-delight/blob/master/CUSTOMIZING.md#custom-scss)) with the following lines (from the exemplary `_custom.scss` file):

https://github.com/kdevo/osprey-delight/blob/9b1fea2b45dc73e9ee45b477ea3011cb879de510/assets/sass/_custom.scss#L35-L42

I'd suggest to follow the instructions in the docs, copy over the lines (without comment chars `/*` and `*/`) and you should be good to go. 

If you like the project, feel free to give it a star! Helps others finding it better.

**Reply by meetmarvin** [reply]
Thank you @kdevo 
I also gave this project a star. Have a good day!
